### PR TITLE
Fix dependencies in the sbt build.

### PR DESCRIPTION
**Pull request by gianm**
SBT needs jackson to be forced or else it writes out a bad pom.

See also https://groups.google.com/d/topic/druid-development/CREoK7o1Z-E/discussion

### CODE REVIEW

File: core/src/test/scala/com/metamx/tranquility/test/DruidBeamTest.scala [hunk 1]
```diff
@@ -131,7 +131,7 @@ class DruidBeamTest extends FunSuite with Matchers
           }
         }
       }
-    ).forType(classOf[Task])
+    ).withType(classOf[Task])
```
Comment (drcrallen) [inline core/src/test/scala/com/metamx/tranquility/test/DruidBeamTest.scala]: is this an intended change in this PR also?

Comment (gianm) [inline core/src/test/scala/com/metamx/tranquility/test/DruidBeamTest.scala]: yes, `forType` is not in jackson 2.4

**Comment by xvrl** [comment]
:+1:

**Comment by drcrallen** [comment]
:+1: after travis

**Pull request merged**